tests/unit/utils/sort-object-on-property-test.js

```diff
@@ -0,0 +1,92 @@
+import { sortObjectsOnProperty } from 'frontend-form-builder/utils/sort-object-on-property';
+import { module, test } from 'qunit';
+
+module('Unit | Utility | sortObjectsOnProperty', function () {
+  module('can sort', function () {
+    test('ascending on property label', function (assert) {
```

Comment:
Only tested on property labe;. Maybe it is good to also test this with numbers and date's
(Sort on numbers is used in the code already and is visually working)